.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+s3Config.json
+sdcData.csv
+sdc-postgresql-data.csv
+sdc-cassandra-data.csv
+sdc-sample-postgresql-data.csv
+sdc-sample-cassandra-data.csv
```

Comment:
not sure if it applies, but you could consider doing all csv files
`*.csv`

Reply:
Thanks, Robin! It does indeed apply and I'll definitely do that.